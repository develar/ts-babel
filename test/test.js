const { generate } = require("../out/JsDocGenerator")
const { transpilePaths } = require("../out/util")
const path = require("path")

test("interface from namespace", () => {
  return transpilePaths([path.join(__dirname, "fixtures")], (basePath, config, tsConfig) => {
    const data = generate(basePath, config, tsConfig, "test", null)
    expect(data.moduleNameToResult).toMatchSnapshot()
    return Promise.resolve(data)
  })
});