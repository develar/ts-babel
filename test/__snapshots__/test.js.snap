// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`interface from namespace 1`] = `
Map {
  "test/out/file" => Array [
    "/**
 * @param {Electron:RequestOptions} options
 * @param {Error} error
 * @param {module:publish-api.VersionInfo} info
 */
foo() {}
",
  ],
}
`;
